Libraries/Components/ActivityIndicator/ActivityIndicator.js

```diff
@@ -29,44 +30,38 @@ type IndicatorSize = number | 'small' | 'large';
 
 type IOSProps = $ReadOnly<{|
   /**
-   * Whether the indicator should hide when not animating (true by default).
-   *
-   * See https://reactnative.dev/docs/activityindicator.html#hideswhenstopped
-   */
+    Whether the indicator should hide when not animating.
+
+    @platform ios
+  */
```

Comment:
I think this is missing an annotation for the default value (`true`).

Reply:
In this case, the default value is coming from FlowTypes via react-docgen.
![Screenshot 2020-08-01 at 5 20 06 PM](https://user-images.githubusercontent.com/31184591/89101209-61e7ad00-d41b-11ea-9496-ff30d77b6c0c.png)